src/main/game/Moves/Moves.java

```diff
@@ -13,4 +14,12 @@ public static Moves getRandomMove() {
         int randomMove = random.nextInt(moves.length);
         return moves[randomMove];
     }
+
+    public static Moves getMove(String move) {
+        HashMap<Object, Object> moves = new HashMap<>();
```

Comment:
As @felipesere suggested above, I think it would be better to be more specific about the types for the generic here, so `HashMap<String, Moves>`

Reply:
Also, you could declare a more generic type that implements the same interface of an `HashMap` for your collection, so instead of `HashMap` on the left hand side just `Map`, and then initialize it as a `HashMap` on the right hand side. This way, if you want to change the type of collection with same interface you only need to do this once (on the right hand side part). I hope this is clear enough!

Reply:
Thank @Gabbendorf - sounds good! I have changed this.